cranelift/filetests/src/test_zkasm.rs

```diff
@@ -237,6 +240,62 @@ mod tests {
         program.join("\n")
     }
 
+    fn run_wat_file(path: &Path) -> Result<(), Box<dyn std::error::Error>> {
+        let engine = Engine::default();
+        let wat = std::fs::read_to_string(path).unwrap();
+        let pat = r#"(import "env" "assert_eq" (func $assert_eq (param i32) (param i32)))"#;
+        let assert_type = if wat.contains(pat) { 32 } else { 64 };
+        let binary = wat::parse_file(path)?;
+        let module = Module::new(&engine, &binary)?;
+        if assert_type == 64 {
+            // 4 a magic number which we must provide for Store initialization, but not important in our case
+            // a searched ways to use some default numbers, but unsuccesfully, so took a number same as in
+            // example in docs
+            // TODO: find a way to do it without magic numbers
+            let mut store = Store::new(&engine, 4);
+            let host_func = Func::wrap(&mut store, |_: Caller<'_, i64>, a: i64, b: i64| {
+                assert_eq!(a, b);
+            });
+            let _instance = Instance::new(&mut store, &module, &[host_func.into()])?;
```

Comment:
Might be worth noting that for many (or even all?) processed WAT files instantiate is sufficient to run the modules $main function since these files have (start $main).

Reply:
I think the regular use case is calling a function exported by a module, as shown in the example. So someone might wonder why there is nothing like `hello.call(...)` here and instead `linker.instantiate(...)` is sufficient. For them a comment explaining that we rely on the `start` function might be helpful.